tx.go

```diff
@@ -274,7 +288,7 @@ func (tx *Tx) Copy(w io.Writer) error {
 func (tx *Tx) WriteTo(w io.Writer) (n int64, err error) {
 	// Attempt to open reader directly.
 	var f *os.File
-	if f, err = os.OpenFile(tx.db.path, os.O_RDONLY|odirect, 0); err != nil {
+	if f, err = os.OpenFile(tx.db.path, os.O_RDONLY|tx.WriteToFlag, 0); err != nil {
 		// Fallback to a regular open if that doesn't work.
```

Comment:
Can you remove this fallback since the user setting O_DIRECT would know whether it is supported on their system or not.